packages/esm-appointments-app/src/patient-queue/visit-form/visit-form.component.tsx

```diff
@@ -98,10 +98,12 @@ const VisitForm: React.FC<VisitFormProps> = ({ patientUuid, appointment }) => {
 
       const payload: NewVisitPayload = {
         patient: patientUuid,
-        startDatetime: toDateObjectStrict(
-          toOmrsIsoString(
-            new Date(dayjs(visitDate).year(), dayjs(visitDate).month(), dayjs(visitDate).date(), hours, minutes),
-          ),
+        startDatetime: new Date(
+          dayjs(visitDate).year(),
+          dayjs(visitDate).month(),
+          dayjs(visitDate).date(),
+          hours,
+          minutes,
         ),
```

Comment:
It might be easier to write this as:

```suggestion
        startDatetime: dayjs(visitDate).set('hour', hours).set('minute', minutes).startOf('minute').toDate()
```